### PR TITLE
Refactor Split Data Module

### PR DESCRIPTION
**Pull request by roshankern**
This PR is ready for review!

In this PR, the split data module is changed to load in `labeled_data` instead of `training_data` for clarity. See #18 for more information.

The index of labeled data is also changed from `index` to `labeled_data_index` to clarify what the index is for.

### CODE REVIEW

File: 1.split_data/scripts/nbconverted/split_data.py [hunk 1]
```diff
@@ -25,27 +24,29 @@
...
 
 # In[3]:
 
 
+# ratio of data to be reserved for testing (ex 0.15 = 15%)
+test_ratio = 0.15
+
 # test_data is pandas dataframe with test split, stratified by Mitocheck_Phenotypic_Class
-test_data = training_data.groupby("Mitocheck_Phenotypic_Class", group_keys=False).apply(
+testing_data = labeled_data.groupby("Mitocheck_Phenotypic_Class", group_keys=False).apply(
```
Comment (gwaybio) [inline 1.split_data/scripts/nbconverted/split_data.py]: you probably should be setting a seed at the beginning of this notebook (sorry if you are already!)

Also, why not use sklearn.model_selection.train_test_split? https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html

File: 1.split_data/scripts/nbconverted/split_data.py [hunk 2]
```diff
@@ -54,10 +55,13 @@
...
 for index in test_indexes:
-    index_data.append({"label": "test", "index": index})
+    index_data.append({"labeled_data_index": index, "label": "test"})
 for index in train_indexes:
-    index_data.append({"label": "train", "index": index})
+    index_data.append({"labeled_data_index": index, "label": "train"})
 index_data = pd.DataFrame(index_data)
+# put indexes into sorted order
+index_data = index_data.sort_values(["labeled_data_index"])
+
```
Comment (gwaybio) [inline 1.split_data/scripts/nbconverted/split_data.py]: consider chaining line 62 and 63:

```python
# I think this works
index_data = (
    pd.DataFrame(index_data)
    .sort_values(["labeled_data_index"])
)
```

Chaining is very pandas

Comment (gwaybio) [reply 1.split_data/scripts/nbconverted/split_data.py]: (`dplyr` also chains really elegantly)

**Review by gwaybio — APPROVED** [review]
LGTM! a couple comments that you _probably_ want to address prior to merging

File: 1.split_data/scripts/nbconverted/split_data.py (hunk 1 above)
Comment (roshankern) [reply 1.split_data/scripts/nbconverted/split_data.py]: I am not sure why I wasn't using this... but I added it in https://github.com/WayScience/phenotypic_profiling_model/pull/19/commits/2f50b408e3448251863eb5497b50cceeb66159c2 with `random_state` (so no seed needed).

File: 1.split_data/scripts/nbconverted/split_data.py (hunk 2 above)
Comment (roshankern) [reply 1.split_data/scripts/nbconverted/split_data.py]: Added in https://github.com/WayScience/phenotypic_profiling_model/pull/19/commits/2f50b408e3448251863eb5497b50cceeb66159c2
